File: tasks/readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>
#include <math.h>

char excl[] = {'.','!','?'};

int main(void)
{
    string text = get_string("Text: ");
    int letter = 0;
    int word = 1;
    int satz = 0;

    for (int i = 0; i < strlen(text); i++)
    {
        if (isalpha(text[i]))  // counting letters with isalpha
        {
            letter++;
        }
        else if (text[i] == ' ') // counting spaces
        {
            word++;
        }
        else if (text[i] == excl[0] || text[i] == excl[1] || text[i] == excl[2]) //I wanned try with array
        {
            satz++;
        }
    }

    float L = (float) letter / (float) word * 100;
    float S = (float) satz / (float) word * 100;

    int index = round(0.0588 * L - 0.296 * S - 15.8);  // Coleman-Liau index

    if (index < 1)
    {
        printf("Before Grade 1\n");
    }
    else if (index > 16)
    {
        printf("Grade 16+\n");
    }
    else
    {
        printf("Grade %i\n", index);
    }
}

File: tasks/sentimental-readability/readability.py
from cs50 import get_string

i = 0
letter = 0
word = 1
satz = 0
excl = [".","!","?"]

text = get_string("Type text: ")

while i < len(text):

    if (text[i].isalpha()):

        letter += 1
        i += 1

    elif (text[i] == " "):

        word += 1
        i +=1

    elif (text[i] == excl[0] or text[i] == excl[1] or text[i] == excl[2]):

        satz += 1
        i += 1
    else:
        i += 1

L = letter / word * 100
S = satz / word * 100

index = round(0.0588 * L - 0.296 * S - 15.8)


if (index < 1):
    print("Before Grade 1")

elif (index >16):
    print("Grade 16+")

else:
    print(f"Grade {index}")